components/allocator/tracking.rs

```diff
@@ -155,6 +157,17 @@ impl<A> AccountingAlloc<A> {
         // the existing allocation data.
         IN_ALLOCATION.with(|status| status.set(true));
         {
+            let Ok(value) = std::env::var("UNTRACKED_LOG_FILE") else {
```

Comment:
Does this work as expected in multiprocess mode, or could we get issues with concurrent writes to the same file?

Reply:
I've pushed a new revision that makes this more clear.